React/UIUtils/RCTUIUtils.m

```diff
@@ -72,18 +72,20 @@ RCTDimensions RCTGetDimensions(RCTPlatformView *rootView) {
   };
   NSScreen *screen = nil;
   NSWindow *window = nil;
+  NSSize windowSize;
```

Comment:
Since this is sometimes window size and sometimes not, can we name it something that doesn't suggest it is always the windowSize?

Reply:
maybe even just "size"